server/src/converter.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs');
+const path = require('path');
+const puppeteer = require('puppeteer');
+const handlebars = require('handlebars');
```

Comment:
Not sure how we want to do this - but the reason I added babel was so we could use things like es6 imports instead of the require statements ... so we might have some inconsistencies with code formats - don't know if that's a huge deal, but thought I'd mention it

Reply:
Changed to import to be consistent, though this caused some problems which I had to solve via adding type: module to package.json and changing some file extensions. Without the fixes I applied I was not able to load any modules, see error below:

import fs from 'fs';
^^^^^^

SyntaxError: Cannot use import statement outside a module